subgraph-js/src/types.ts

```diff
@@ -60,6 +61,58 @@ function addTypeNameToPossibleReturn<T>(
   return maybeObject as null | (T & { __typename: string });
 }
 
+export function entitiesResolver({
```

Comment:
Is there any functional change in this file that I'm not seeing or is it just a non-function refactor?

Reply:
There is not change to the resolver code, but it extract it as a method because it's called in `buildSubgraphSchema` directly now.